### ISSUE TITLE
Started to get a lot of crashes when looking up travel details

### ISSUE THREAD

**Issue by thyberg** [johannilsson/sthlmtraveling#83]
I managed to get the following stacktrace out of my phone:

```02-24 11:16:43.958 W/ResourceType(19198): No known package when getting value for resource number 0x020201d7
02-24 11:16:43.960 E/AndroidRuntime(19198): FATAL EXCEPTION: main
02-24 11:16:43.960 E/AndroidRuntime(19198): Process: com.markupartist.sthlmtraveling, PID: 19198
02-24 11:16:43.960 E/AndroidRuntime(19198): java.lang.NullPointerException: Attempt to invoke virtual method 'int android.graphics.Bitmap.getWidth()' on a null object reference
02-24 11:16:43.960 E/AndroidRuntime(19198): 	at com.android.webview.chromium.Ap.getDefaultVideoPoster(WebViewContentsClientAdapter.java:578)
02-24 11:16:43.960 E/AndroidRuntime(19198): 	at org.chromium.android_webview.DefaultVideoPosterRequestHandler$$Lambda$0.run(Unknown Source:2)
02-24 11:16:43.960 E/AndroidRuntime(19198): 	at android.os.Handler.handleCallback(Handler.java:789)
02-24 11:16:43.960 E/AndroidRuntime(19198): 	at android.os.Handler.dispatchMessage(Handler.java:98)
02-24 11:16:43.960 E/AndroidRuntime(19198): 	at android.os.Looper.loop(Looper.java:164)
02-24 11:16:43.960 E/AndroidRuntime(19198): 	at android.app.ActivityThread.main(ActivityThread.java:6809)
02-24 11:16:43.960 E/AndroidRuntime(19198): 	at java.lang.reflect.Method.invoke(Native Method)
02-24 11:16:43.960 E/AndroidRuntime(19198): 	at com.android.internal.os.Zygote$MethodAndArgsCaller.run(Zygote.java:240)
02-24 11:16:43.960 E/AndroidRuntime(19198): 	at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:767)
02-24 11:16:43.970 I/vendor.qti.bluetooth@1.0-ibs_handler(  760): ProcessIbsCmd: Received IBS_SLEEP_IND: 0xFE
02-24 11:16:43.970 I/vendor.qti.bluetooth@1.0-ibs_handler(  760): SerialClockVote: vote for UART CLK OFF
02-24 11:16:43.971 I/MediaCodec(19198): name: video/avc```

It seems to happen when the app is trying to show an ad.

**Comment by thyberg** [comment]
It seems to have been related to a certain ad, since I only happened for a few minutes.
I am unable to recreated it now when I am trying to record the steps taken using vysor.

**Comment by johannilsson** [comment]
Thanks for the report. This crash is related to a bug in the library used for showing ads. It requires an update of the library to be resolved. I thought I blocked the ads that caused this, but apparently not. I've added additional filters on the ad backend but will roll out an update with the potential fix as well.

If this crash is still happening it's possible to turn off the ads through settings.

Thanks.
